main.go

```diff
@@ -275,6 +276,9 @@ func main() {
 		}),
 		Cache: cache.Options{
 			DefaultNamespaces: namespaces,
+			DefaultLabelSelector: labels.SelectorFromSet(map[string]string{
```

Comment:
I followed the docs from the [operator-sdk](https://sdk.operatorframework.io/docs/best-practices/designing-lean-operators/) but they're out of date after this release https://github.com/kubernetes-sigs/controller-runtime/pull/2300. This is the best way going forward!